test/korma_encrypted/core_test.clj

```diff
@@ -46,14 +46,15 @@
     (sql/db-do-commands
        spec
        (tap (sql/create-table-ddl "credit_cards"
-                                  [:id "serial primary key"]
```

Comment:
Seems to me like we likely need to allow configuring the name of the primary key column.